io/BpfWriter.hpp

```diff
@@ -56,6 +56,9 @@ class PDAL_DLL BpfWriter : public FlexWriter
         CoordId() : m_auto(false), m_val(0)
         {}
 
+        CoordId(bool isAuto, int val) : m_auto(isAuto), m_val(val)
```

Comment:
AFAICT, you don't need this or the header change. You can just change `m_auto` to `true` in the above ctor and that's it.

Reply:
My compiler complained when I tried to do that without making a ctor.

Reply:
Where was that code?

Reply:
Neither of these work without making the ctor
```
args.add("coord_id", "UTM coordinate ID", m_coordId, "auto");
```

```
args.add("coord_id", "UTM coordinate ID", m_coordId, {true, 0});
```

Reply:
Just drop the last argument.

Reply:
> And change the `false` to `true` in the default constructor.

I didn't think we wanted to do that because it would mean finding all usages of CoordId and making sure that auto wasn't set to `true` when we were using it.